utils/vara-ui/src/components/button/helpers.ts

```diff
@@ -0,0 +1,8 @@
+const buttonSizes = ['x-small', 'small', 'medium', 'default', 'x-large'] as const;
+type IButtonSizes = (typeof buttonSizes)[number];
+
+const buttonColors = ['primary', 'dark', 'light', 'grey', 'border', 'transparent', 'destructive'] as const;
```

Comment:
`error` or `warning` more descriptive than `destructive`

Reply:
Most common is `danger`